documentation/api/v1/auth_api_doc.rb

```diff
@@ -0,0 +1,66 @@
+require 'swagger/blocks'
+
+module AuthApiDoc
+  include Swagger::Blocks
+
+  MESSAGE_DESC = 'The status of the authentication request.'
+  MESSAGE_EXAMPLE = 'Generated new API token.'
+  TOKEN_DESC = 'The Authentication Bearer token'
+  TOKEN_EXAMPLE = '899d2f45e12429d07427230289400a4594bcffe32169ebb826b4ffa9b90e1d1586f15fa42f069bb7'
+
+  # Swagger documentation for auth model
+  swagger_schema :Auth do
+    property :message, type: :string, description: MESSAGE_DESC, example: MESSAGE_EXAMPLE
+    property :token, type: :string, description: TOKEN_DESC, example: TOKEN_EXAMPLE
+  end
+
+  swagger_path '/api/v1/auth/generate-token' do
+    # Swagger documentation for /api/v1/auth/generate-token GET
+    operation :get do
+
+      key :description, 'Return a valid Authorization Bearer token.'
+      key :tags, [ 'auth' ]
+
+      parameter do
+        key :name, :username
+        key :in, :query
+        key :description, ''
```

Comment:
I know it's pretty self-explanatory, but can we add a basic description for these properties?